bundles/org.openhab.binding.hdpowerview/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -75,13 +75,40 @@
 		<representation-property>id</representation-property>
 
 		<config-description>
-			<parameter name="id" type="text" required="true">
+			<parameter name="id" type="integer" min="1" required="true">
```

Comment:
Is this backward compatible?

Reply:
Yes, but hidden as last sentence in the PR description: 
> I have tested backwards compatibility for the change by creating a thing with version 3.2 of the binding from UI, then migrated to the PR version and verified that thing still works correctly.